source/install/verify-signatures.txt

```diff
@@ -0,0 +1,176 @@
+      following output:
+
+      .. code-block:: sh
+         :copyable: false
+
+         gpg: key CEED0419D361CB16: public key "Mongosh Release Signing Key <packaging@mongodb.com>" imported
+         gpg: Total number processed: 1
+         gpg:               imported: 1
+
+      If you have previously imported the key, the command returns the
+      following output:
+
+      .. code-block:: sh
+         :copyable: false
+
+         gpg: key A8130EC3F9F5F923: "Mongosh Release Signing Key <packaging@mongodb.com>" not changed
+         gpg: Total number processed: 1
+         gpg:             	      unchanged: 1
+
+   .. step:: Download the MongoDB Shell public signature
+
+      To download the MongoDB Shell public signature, go to the `mongosh
+      Releases <https://github.com/mongodb-js/mongosh/releases>`__ page
+      on GitHub and download the corresponding ``.sig`` file for your
+      version and variant.
+
+      For example, if you are running
+      ``mongosh-{+version+}-linux-arm64``, download
+      ``mongosh-{+version+}-linux-arm64.sig``
+
+      .. note:: 
+         
+         Make sure that you select the correct version in the GitHub
+         releases page when you download the signature.
+
+   .. step:: Verify the package
+
+      Run this command:
+
+      .. code-block:: sh
+
+         gpg --verify <path_to_signature_file> <path_to_mongosh_executable>
+
+      If the package is properly signed by MongoDB, the command returns
+      this response:
+      
+      .. code-block:: sh
+         :copyable: false
+
+         gpg: Signature made Mon Jan 22 10:22:53 2024 CET
+         gpg:                using RSA key AB1B92FFBE0D3740425DAD16A8130EC3F9F5F923
+         gpg: Good signature from "Mongosh Release Signing Key <packaging@mongodb.com>" [unknown]
+         
+      If the package is properly signed but you do not trust the signing
+      key in your local ``trustdb``, ``gpg`` returns this message:
+
+      .. code-block:: sh
+         :copyable: false
+
+         gpg: WARNING: This key is not certified with a trusted signature!
+         gpg:          There is no indication that the signature belongs to the owner.
+         
+      If the package is not signed properly, the command returns an
+      error message:
+
+      .. code-block:: sh
+         :copyable: false
+
+         gpg: Signature made Mon Jan 22 10:22:53 2024 CET
+         gpg:                using RSA key AB1B92FFBE0D3740425DAD16A8130EC3F9F5F923
+         gpg: BAD signature from "Mongosh Release Signing Key <packaging@mongodb.com>" [unknown]
+
+.. _verify-signatures-rpm:
+
+Verify RPM Packages for RedHat
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. procedure::
+   :style: normal
+
+   .. step:: Import the MongoDB Shell public key in gpg and rpm
+
+      Run this command:
+
+      .. code-block:: sh
+
+         curl https://pgp.mongodb.com/mongosh.asc | gpg --import rpm --import https://pgp.mongodb.com/mongosh.asc
```

Comment:
these are two different commands. first user needs to import it in gpg and then in rpm.

```suggestion
         curl https://pgp.mongodb.com/mongosh.asc | gpg --import
         rpm --import https://pgp.mongodb.com/mongosh.asc
```